app-header-layout/demo/demo2.html

```diff
@@ -63,10 +68,10 @@
 
   <!-- app-header-layout provides a scroller -->
   <app-header-layout has-scrolling-region>
-    <app-header condenses reveals>
+    <app-header effects="waterfall" condenses reveals>
       <app-toolbar>
         <paper-icon-button icon="menu"></paper-icon-button>
-        <div class="flex"></div>
+        <div title></div>
```

Comment:
Can you also remove the [classes/iron-flex-layout.html import](https://github.com/PolymerLabs/app-layout/blob/header-snap/app-header-layout/demo/demo2.html#L22)?